duckduckgo/Android
# Delete old donotsell.js

Task/Issue URL: https://app.asana.com/0/414730916066338/1201164941418932

### Description
This PR deletes the old donotsell.js file which is not used anymore, instead we use the gpc.js file in the privacy-config module

### Steps to test this PR
1. Visit test page and ensure that Client-side detection is passing: `https://global-privacy-control.glitch.me/`
1. Toggle GPC off and check that Client-side detection is no longer passing.
